mimic/model/glance_objects.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
Ha, good point. I dont want to address them in this PR, but will create issues for them and address them separately.